### DISCUSSION TITLE
Is there a way to create and access multiple sqlite database

### DISCUSSION THREAD

**Discussion by 1dveeran** [SeaQL/sea-orm#1195] [Q&A] [answered]
1. I want to create sqlite database with sea-orm for service apps, is it possible?
2. I am trying to use sea-orm for creating multiple Sqlite database and access them. But i getting issue with the default environment variable "DATABASE_URL". Is there a way to change this.
e.g.
DATABASE_URL='sqlite://db1.db?mode=rwc'
DATABASE_URL1='sqlite://db2.db?mode=rwc'
DATABASE_URL3='sqlite://db3.db?mode=rwc'

**Comment by billy1624** [comment, marked as answer]
Hey @1dveeran, sorry for the delay. This is achievable, you need to create a connection for each database. 

```rs
let db1: DatabaseConnection = Database::connect("sqlite://db1.db?mode=rwc").await?;
let db2: DatabaseConnection = Database::connect("sqlite://db2.db?mode=rwc").await?;
let db3: DatabaseConnection = Database::connect("sqlite://db3.db?mode=rwc").await?;
```
